draft-ietf-tls-exported-authenticator.md

```diff
@@ -185,22 +189,24 @@ present.  If no authenticator request is provided, the certificate_request_conte
 can be chosen arbitrarily.
 
 The certificates chosen in the Certificate message MUST conform to the
-requirements of a Certificate message in the negotiated version of TLS. In
+requirements of a Certificate message in the negotiated version of TLS.  In
 particular, the certificate MUST be valid for the a signature algorithm
-indicated by the peer in a "signature_algorithms" extension, as described in
-Section 4.2.3 of {{!TLS13}} and Sections 7.4.2 and 7.4.6 of {{!RFC5246}}.
+indicated by the peer in a "signature_algorithms_cert" extension, as described in
+Section 4.2.3 of {{!TLS13}} for TLS 1.3 or the "signature_algorithms" extension
+from Sections 7.4.2 and 7.4.6 of {{!RFC5246}} for TLS 1.2.
 
-In addition to "signature_algorithms", the "server_name" {{!RFC6066}},
-"certificate_authorities" (Section 4.2.4. of {{!TLS13}}), or "oid_filters"
+In addition to "signature_algorithms" and "signature_algorithms_cert",
+the "server_name" {{!RFC6066}}, "certificate_authorities"
+(Section 4.2.4. of {{!TLS13}}), and "oid_filters"
 (Section 4.2.5. of {{!TLS13}}) extensions are used to guide certificate
-selection. These extensions are taken from the authenticator request if
+selection.  These extensions are taken from the authenticator request if
```

Comment:
There's a risk here that the list be interpreted as definitive.  That is, that we can't use other extensions.  Maybe in this next sentence include "The extensions, or others that might affect certificate selection, ..." to avoid this misconception.